$MirOS$

	Be able to compile llvm-gcc with itself and -std=gnu99

--- gcc/toplev.h.orig	Thu Jan  1 00:00:00 1970
+++ gcc/toplev.h	Tue Nov 11 02:14:22 2008
@@ -174,13 +174,19 @@ extern int floor_log2                  (
 #  define CTZ_HWI __builtin_ctz
 # endif
 
-extern inline int
+#ifdef __GNUC_STDC_INLINE__
+#define GCC_TOPLEV_H_INLINE	extern inline __attribute((__gnu_inline__))
+#else
+#define GCC_TOPLEV_H_INLINE	extern inline
+#endif
+
+GCC_TOPLEV_H_INLINE int
 floor_log2 (unsigned HOST_WIDE_INT x)
 {
   return x ? HOST_BITS_PER_WIDE_INT - 1 - (int) CLZ_HWI (x) : -1;
 }
 
-extern inline int
+GCC_TOPLEV_H_INLINE int
 exact_log2 (unsigned HOST_WIDE_INT x)
 {
   return x == (x & -x) && x ? (int) CTZ_HWI (x) : -1;
